tests/PHPUnit/Framework/TestCase/UnitTestCase.php

```diff
@@ -18,16 +21,45 @@
  */
 abstract class UnitTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
Nah I don't think you should worry about that I just wanted to make sure we are on the same page as to what the ideal goal is, we can do that later.